models/club.ts

```diff
@@ -0,0 +1,12 @@
+import { DocumentReference, DocumentData } from "firebase/firestore"
+import Contacts from "./contacts"
+import Event from "./event"
+
+interface Club {
```

Comment:
Just a suggestion...maybe put all your models in the same file or colocate your models with your query methods. A lot of these model files are very small so might be easier to condense it in a single place.

Reply:
i see, i prefer them as multiple files. don't they get optimized away by webpack or some similar technology?